### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Hi... I have a handful of spelling fixes as [identified by a tool I work on](https://github.com/jsoref/FCPCafe/actions/runs/11790095462#summary-32839979148) and a larger (but admittedly incomplete) set of accessibility fixes that I'd hope you'd consider.

I'm happy to split them up in some other manner if that's helpful.

### CODE REVIEW

File: docs/_includes/bugtracker-previous.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - [Internal Stabilisation issues and resulting crashes (January 17, 2024)](https://github.com/CommandPost/FCPCafe/issues/326)
 - [Missing smooth position keyframes in FCPXML (January 8, 2024)](https://github.com/CommandPost/FCPCafe/issues/321)
 - [File Storage Issue when Transferring Projects between Libraries (January 3, 2024)](https://github.com/CommandPost/FCPCafe/issues/318)
-- [Can't collapse or uncollapse Parameter SubGroup's programatically  (December 29, 2023)](https://github.com/CommandPost/FCPCafe/issues/315)
+- [Can't collapse or uncollapse Parameter SubGroup's programmatically  (December 29, 2023)](https://github.com/CommandPost/FCPCafe/issues/315)
```
Comment (jsoref) [inline docs/_includes/bugtracker-previous.md:15]: This is a more classical spelling correction

File: docs/_includes/news/20241030.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ We [already knew from WWDC](/news/20240611/) that:
 
 ...so we knew that Spatial Video features were already on the cards, but we got some new surprises in the Mac mini video:
 
-![](/static/mac-mini-annoucement.jpg)
+![](/static/mac-mini-announcement.jpg)
```
Comment (jsoref) [inline docs/_includes/news/20241030.md:13]: this file is being renamed too

File: docs/case-studies/women-of-resilience.md [hunk 3]
```diff
@@ -184,7 +184,7 @@ Remember the captions? The only thing not speed-changeable (why not, I ask, woul
 
 _**Screenshots 8A-B:** This way I can speed change (adjust timings) all SRTs too for re-import._
 
-Let’s not forget to, again, add the Broadcast Safe to the clean-feed to make it ‘legal’. We already know hot to proceed with exporting Multitrack Quicktime files. We are done and have exported several versions which usually takes a lot longer and additional people.
+Let’s not forget to, again, add the Broadcast Safe to the clean-feed to make it ‘legal’. We already know hot to proceed with exporting Multitrack QuickTime files. We are done and have exported several versions which usually takes a lot longer and additional people.
```
Comment (jsoref) [inline docs/case-studies/women-of-resilience.md:187]: brand...

File: docs/case-studies/women-of-resilience.md [hunk 4]
```diff
@@ -194,7 +194,7 @@ _**Phew... I think we all need a holiday, right?**_
...
 
-If you are interested in using my special (ported from Motion) High-Pass Sharpening filter, which is better than the native unsharp mask sharpening (yuck!), you can download it [here](www.sebastianleitner.com/highpass-fcp.zip). **UPDATE March 20th, 2024:** It appears the native sharpening effect got a lot better with one of the latest FCP updates and behaves more like a smart sharpen frequency filter now. Cool! You can actually use that now.
+If you are interested in using my special (ported from Motion) High-Pass Sharpening filter, which is better than the native unsharp mask sharpening (yuck!), you can [download it](https://www.sebastianleitner.com/highpass-fcp.zip). **UPDATE March 20th, 2024:** It appears the native sharpening effect got a lot better with one of the latest FCP updates and behaves more like a smart sharpen frequency filter now. Cool! You can actually use that now.
```
Comment (jsoref) [inline docs/case-studies/women-of-resilience.md:197]: Markdown doesn't tend to automatically prefix `https://` for links...

File: docs/cut-on-fcp.md [hunk 5]
```diff
@@ -113,7 +113,7 @@ Here's some other success stories:
 - Ripples Of Life (2021) {{fcpx}} {{imdb}}(https://www.imdb.com/title/tt13736078/)
 - The Business of Christmas 2 (2021) {{fcpx}} {{imdb}}(https://www.imdb.com/title/tt14662334/)
 - Why? (2021) {{fcpx}} {{imdb}}(https://www.imdb.com/title/tt10313806/)
-- The Portait of a Nightmare (2020) {{fcpx}} {{imdb}}(https://www.imdb.com/title/tt11853706/)
+- The Portrait of a Nightmare (2020) {{fcpx}} {{imdb}}(https://www.imdb.com/title/tt11853706/)
```
Comment (jsoref) [inline docs/cut-on-fcp.md:116]: I checked the title of this item...

File: docs/openletter.md [hunk 6]
```diff
@@ -114,7 +114,7 @@ Signed,
 42. Tanneguy O'Meara, Editor of ‘Gerard Depardieu : Mon Reve Ouzbek,’ Feature Documentary, France
 43. Helgi Thor, Editor of ‘Cine Holliúdy 2: A Chibata Sideral ,’ Feature Film, Brazil
 44. Riffaud Richard, Editor of ‘Cobweb,’ Feature Film, US
-45. Stephen Nahorniak, Editor of ‘Open Hous,’ TV Reality, TV Documentary, TV Non-Fiction, US
+45. Stephen Nahorniak, Editor of ‘Open House,’ TV Reality, TV Documentary, TV Non-Fiction, US
```
Comment (jsoref) [inline docs/openletter.md:117]: I checked the title of this too

File: docs/update-guide.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ Unfortunately when it comes to apps on the Apple App Store, you can only ever do
 
 This means, if you're a professional Final Cut Pro editor, you need to be very careful about keeping copies of older versions of Final Cut Pro, Compressor and Motion as backups incase there's bugs, but also incase you have to access older Libraries in the future that may not open in the latest version of Final Cut Pro.
 
-You can learn more about backing up Final Cut Pro on Apple's site [here](https://support.apple.com/en-au/HT203010).
```
Comment (jsoref) [inline docs/update-guide.md:9]: I've only removed the `/en-.../` items from a couple of links, but in general you don't really need them as Apple will redirect to the appropriate region based on the viewer.

File: docs/update-guide.md [hunk 8]
```diff
@@ -78,7 +78,7 @@ If you use **MotionVFX**, mLUT, mFlare 2, mFilmLook, mMuppet & mTracker all requ
 
 If you can't use your camera or video output device after updating to **macOS Sonoma 14.1**, Apple has released a workaround.
 
-Starting in **macOS Sonoma 14.1**, cameras and video output devices that don't use modern system extensions won't be available to use unless you restore the legacy settings. You can learn more [here](https://support.apple.com/en-us/HT213969).
```
Comment (jsoref) [inline docs/update-guide.md:81]: another link, different locale.

**Comment by latenitefilms** [comment]
Thanks! This week is a bit of a crazy week, but will try have a proper look and think about this soon!

**Pull request merged**